site/docs/components/button/usage.mdx

```diff
@@ -9,14 +11,21 @@ data:
 
 ### Using the Button component
 
-- Keep the text label short. Ideally one to three words and no more than five. If you’re including an icon in the label, use no more than three words.
+- Keep the text description short, ideally one to three words and no more than five. If you’re including an icon with the text, use no more than three words.
+- For icons, use the Salt Figma icon library. If the icon you need isn’t available, follow the steps in our [Creating Salt icons](<https://www.figma.com/file/ibGX0b0CxUcZU3pn6OrqX7/Salt%3A-Icons-(Community)?type=design&node-id=490-74&mode=design>) guide.
 
-- When including icons, use the Salt Figma icon library. If the icon you need isn’t available, follow the steps in our Creating Salt icons guide.
+#### When to use Button
 
-- Consider a Link instead of the Button component if the user is navigating to a new page or window rather than carrying out an action.
+- To allow the user to execute an action, such as “submit”, “merge” or “upload”. Buttons are typically placed in components and contexts such as [Dialog](../dialog), [Cards](../cards) and [Forms](/salt/patterns/forms).
```

Comment:
```suggestion
- To allow the user to execute an action, such as “submit”, “merge” or “upload”. Buttons are typically placed in components and contexts such as [Dialog](../dialog), [Card](../card) and [Form](/salt/patterns/form).
```

Reply:
Will update for card and keep the same for forms (https://www.saltdesignsystem.com/salt/patterns/forms)